core/dronecore.h

```diff
@@ -53,14 +53,31 @@ class DroneCore
         DEVICE_BUSY, /**< @brief %Device is busy. */
         COMMAND_DENIED, /**< @brief Command is denied. */
         DESTINATION_IP_UNKNOWN, /**< @brief %Connection IP is unknown. */
-        CONNECTIONS_EXHAUSTED /**< @brief %Connections exhausted. */
+        CONNECTIONS_EXHAUSTED, /**< @brief %Connections exhausted. */
+        CONNECTION_URL_INVALID /**< @brief URL invalid. */
     };
 
     /**
      * @brief Translates the DroneCore::ConnectionResult enum to a human-readable English string.
      */
     static const char *connection_result_str(ConnectionResult);
 
+    /**
+     * @brief Adds Connection via URL
+     *
+     * Supports connection: Serial, TCP or UDP.
+     * Connection URL format should be:
+     * - UDP - udp://[Bind_host][:Bind_port]
+     * - TCP - tcp://[Server_host][:Server_port]
+     * - Serial - serial://[Dev_Node][:Baudrate]
+     *
+     * Default url : udp://:14540.
```

Comment:
Do we need to say that the default Bind_host is localhost? It isn't specified in the URL and I think this is fairly obvious ... but might be more clear if it was.